apps/src/javalab/javalabRedux.js

```diff
@@ -4,13 +4,14 @@ const APPEND_CONSOLE_LOG = 'javalab/APPEND_CONSOLE_LOG';
 const RENAME_FILE = 'javalab/RENAME_FILE';
 const SET_SOURCE = 'javalab/SET_SOURCE';
 const SOURCE_VISIBILITY_UPDATED = 'javalab/SOURCE_VISIBILITY_UPDATED';
+const SOURCE_VALIDATION_UPDATED = 'javalab/SOURCE_VALIDATION_UPDATED';
 const SET_ALL_SOURCES = 'javalab/SET_ALL_SOURCES';
 const COLOR_PREFERENCE_UPDATED = 'javalab/COLOR_PREFERENCE_UPDATED';
 const REMOVE_FILE = 'javalab/REMOVE_FILE';
 
 const initialState = {
   consoleLogs: [],
-  sources: {'MyClass.java': {text: '', visible: true}},
+  sources: {'MyClass.java': {text: '', visible: true, isValidationFile: false}},
```

Comment:
let's align these boolean namings. I don't think we need to specify `file` in the name, since it's always metadata on a file, and I don't think we have picked up `visible` on the javabuilder side, so we can either set them to `visible` and `validation` or switch to `isVisible`, `isValidation` if we want to be more clear that they are booleans and align with the variable names we are using below.

Reply:
Good point- aligned them all to 'is'.